src/lib.rs

```diff
@@ -8,3 +8,4 @@ pub mod smbc;
 pub use crate::{error::*, smbc::*};
 
 pub use crate::parser::*;
+extern crate rust_smbclient_sys;
```

Comment:
This shouldn't be needed because we're using the 2018 edition rust :)

Reply:
👍